### PR TITLE
OWNERS: Cleanup pt. 2

### PR DESCRIPTION
**Pull request by justaugustus**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

(Continues https://github.com/kubernetes/release/pull/2772.)

- OWNERS(generated): Prune via maintainers tool
- OWNERS: Restore non-critical path approvers removed via generator
- cmd/ci-reporter: Add `ci-signal-reporter-approvers` alias
- OWNERS: Alpha-sort members

The most notable change is that this PR prunes several members of the @kubernetes/release-engineering team, namely, the following Release Manager Associates:

- ~@jimangel~ (expressed interest in staying on in https://github.com/kubernetes/release/pull/2773#issuecomment-1338226879)
- @onlydole
- @sethmccombs
- @wilsonehusin

Note that I've opened this PR "for science" and as a means to frame the discussion.

If you are on the above list and still have the time/interest in continuing as a Release Manager Associate, feel free to comment here to that effect.

I'll also send a note to the release-managers@ list and we can discuss there or via DM, if you'd prefer.

Explicit hold for discussion ~(and to rebase out the commits from https://github.com/kubernetes/release/pull/2772)~:
/hold

Signed-off-by: Stephen Augustus <foo@auggie.dev>

/assign @saschagrunert @jeremyrickard @Verolop @cpanato @puerco 
cc: @kubernetes/release-engineering 

#### Special notes for your reviewer:

The first commit was generated via https://github.com/kubernetes-sigs/maintainers.
The `--exclude` list is @kubernetes/sig-release-leads, who should not be removed from OWNERS, as well as the Release Manager Associates that were recently added in https://github.com/kubernetes/release/pull/2772.

```log
❯ time GOPATH=$(go env GOPATH) maintainers prune \
  --repository-devstats "kubernetes/release" \
  --repository-github "kubernetes/release" \
  --dryrun=false \
  --exclude cpanato,jeremyrickard,justaugustus,puerco,saschagrunert,Verolop,cici37,jrsapi,salaxander
Running script : 11-25-2022 17:27:42
Processed /Users/augustus/prj/k8s/release/OWNERS
Processed /Users/augustus/prj/k8s/release/cmd/ci-reporter/OWNERS
Processed /Users/augustus/prj/k8s/release/cmd/krel/OWNERS
Processed /Users/augustus/prj/k8s/release/cmd/kubepkg/OWNERS
Processed /Users/augustus/prj/k8s/release/cmd/release-notes/OWNERS
Processed /Users/augustus/prj/k8s/release/hack/rapture/OWNERS
Processed /Users/augustus/prj/k8s/release/images/OWNERS
Processed /Users/augustus/prj/k8s/release/images/build/OWNERS
Processed /Users/augustus/prj/k8s/release/images/build/debian-base/OWNERS
Processed /Users/augustus/prj/k8s/release/images/build/debian-iptables/OWNERS
Processed /Users/augustus/prj/k8s/release/images/build/distroless-iptables/OWNERS
Processed /Users/augustus/prj/k8s/release/images/build/go-runner/OWNERS
Processed /Users/augustus/prj/k8s/release/packages/deb/OWNERS
Processed /Users/augustus/prj/k8s/release/packages/rpm/OWNERS
Processed /Users/augustus/prj/k8s/release/pkg/OWNERS
Processed /Users/augustus/prj/k8s/release/pkg/notes/OWNERS
Processed /Users/augustus/prj/k8s/release/OWNERS_ALIASES
Found 9 unique aliases
Found 37 unique users
.........................................


>>>>> Contributions from kubernetes/release devstats repo and kubernetes/release github repo : 25
>>>>> GitHub ID : Devstats contrib count : GitHub PR comment count
cpanato : 452 : 336
saschagrunert : 365 : 167
puerco : 132 : 51
xmudrii : 97 : 32
ameukam : 49 : 33
palnabarun : 83 : 21
Verolop : 16 : 11
leonardpahlke : 107 : 10
justaugustus : 84 : 36
aojea : 14 : 5
onlydole : 3 : 5
danwinship : 16 : 1
jeremyrickard : 18 : 2
wilsonehusin : 18 : 5
robscott : 13 : 12
Nivedita-coder : 7 : 1
cici37 : 3 : 2
BenTheElder : 36 : 4
bharitwal : 2 : 1
bowei : 2 : 1
MushuEE : 2 : 1
sethmccombs : 1 : 1
spiffxp : 2 : 0
jimangel : 2 : 4
BenjaminKazemi : 2 : 1


>>>>> Missing Contributions in kubernetes/release (devstats == 0): 12
Namanl2001
andrewsykim
caseydavenport
dcbw
freehan
jrsapi
khenidak
mehabhalodiya
mrhohn
salaxander
shuheiktgw
thockin


>>>>> Low reviews/approvals in kubernetes/release (GH pr comments <= 10 && devstats <=20): 14
BenjaminKazemi
MushuEE
Nivedita-coder
aojea
bharitwal
bowei
cici37
danwinship
jeremyrickard
jimangel
onlydole
sethmccombs
spiffxp
wilsonehusin
Fixing up /Users/augustus/prj/k8s/release/OWNERS
Fixing up /Users/augustus/prj/k8s/release/cmd/ci-reporter/OWNERS
Fixing up /Users/augustus/prj/k8s/release/cmd/krel/OWNERS
Fixing up /Users/augustus/prj/k8s/release/cmd/kubepkg/OWNERS
Fixing up /Users/augustus/prj/k8s/release/cmd/release-notes/OWNERS
Fixing up /Users/augustus/prj/k8s/release/hack/rapture/OWNERS
Fixing up /Users/augustus/prj/k8s/release/images/OWNERS
Fixing up /Users/augustus/prj/k8s/release/images/build/OWNERS
Fixing up /Users/augustus/prj/k8s/release/images/build/debian-base/OWNERS
Fixing up /Users/augustus/prj/k8s/release/images/build/debian-iptables/OWNERS
Fixing up /Users/augustus/prj/k8s/release/images/build/distroless-iptables/OWNERS
Fixing up /Users/augustus/prj/k8s/release/images/build/go-runner/OWNERS
Fixing up /Users/augustus/prj/k8s/release/packages/deb/OWNERS
Fixing up /Users/augustus/prj/k8s/release/packages/rpm/OWNERS
Fixing up /Users/augustus/prj/k8s/release/pkg/OWNERS
Fixing up /Users/augustus/prj/k8s/release/pkg/notes/OWNERS
Fixing up /Users/augustus/prj/k8s/release/OWNERS_ALIASES
GOPATH=$(go env GOPATH) maintainers prune --repository-devstats     --exclude  0.19s user 0.11s system 0% cpu 2:20.83 total
```

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by onlydole** [comment]
Thank you, @justaugustus! This looks good to me and I liked all the science-y call outs!

**Comment by justaugustus** [comment]
> I'll also send a note to the release-managers@ list and we can discuss there or via DM, if you'd prefer.

Note sent to @kubernetes/release-engineering: https://groups.google.com/a/kubernetes.io/g/release-managers/c/WdHa9JoWrJY/m/O6otQmEMBwAJ

Consensus date set for Friday, December 9th, 3pm US Eastern ([ref](https://groups.google.com/a/kubernetes.io/g/release-managers/c/WdHa9JoWrJY/m/P1O37KoMBwAJ)).

**Comment by justaugustus** [comment]
> Thank you, @justaugustus! This looks good to me and I liked all the science-y call outs!

@onlydole -- Thanks for the quick response and thank you for serving on the team!!

**Comment by justaugustus** [comment]
(Rebased out https://github.com/kubernetes/release/pull/2772.)

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -20,23 +20,23 @@ aliases:
   release-engineering-reviewers:
     - ameukam # Release Manager Associate
     - cici37 # Release Manager Associate
-    - jimangel # Release Manager Associate
     - jrsapi # Release Manager Associate
-    - onlydole # Release Manager Associate
```
Comment (justaugustus) [inline OWNERS_ALIASES:25]: Confirmed: https://github.com/kubernetes/release/pull/2773#issuecomment-1327926443

**Comment by wilsonehusin** [comment]
Confirming that I'd like to resign from my position — I appreciate all the time we got to work together ☸️

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -20,23 +20,23 @@ aliases:
...
     - salaxander # Release Manager Associate
-    - sethmccombs # Release Manager Associate
-    - wilsonehusin # Release Manager Associate
```
Comment (justaugustus) [inline OWNERS_ALIASES:28]: Confirmed: https://github.com/kubernetes/release/pull/2773#issuecomment-1327971516

**Comment by justaugustus** [comment]
> Confirming that I'd like to resign from my position — I appreciate all the time we got to work together ☸️

Thanks for being a Release Manager Associate, @wilsonehusin!!!

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by jimangel** [comment]
👋  Hey all,

I'm interested in continuing as an RMA if there's room to grow into other roles _and_ I'm still providing value to the SIG.

I think there was a missed, _assumed_, expectation this year – after shadowing @palnabarun – that I would lead branch management for the next release. However, I was never formally asked, and I ended up going on my (planned) 12 weeks of paternity leave mid-May.

When I returned (late September), I chatted briefly with @Verolop about helping out from the shadow side (sounds dark 😜) for branch management, but other folks were selected.

I'm involved with the OBS pilot to the extent that I'm in a Slack channel and have a meeting invite, but not really asked to help with anything specific to deliver. It feels like @xmudrii is driving progress solo (and doing a kick-ass job!)

I've reached out to @puerco about helping with SLSA levels too, but lack a clear entrance point to provide impact (seeing heroics of the like of @saschagrunert optimizing call concurrency).

TL;DR: I'm interested in staying an RMA if there are opportunities to provide impact, but I feel excluded / lack direction currently.

Also, no biggie if the answer is "no" and we want to roll with devstats as a measure of interest...

/lgtm

**Comment by sethmccombs** [comment]
All good here - haven't had the time in a while to work with y'all. Hopefully I'll get back to it some day! Cheers <3

**Comment by justaugustus** [comment]
> All good here - haven't had the time in a while to work with y'all. Hopefully I'll get back to it some day! Cheers <3

Thanks for checking in and for being a Release Manager Associate, @sethmccombs!!!

**Comment by justaugustus** [comment]
> I'm interested in continuing as an RMA if there's room to grow into other roles _and_ I'm still providing value to the SIG.

Thanks for checking in, @jimangel!
Restored your entry in https://github.com/kubernetes/release/pull/2773/commits/d5b3b91cc094f5e9783995e2045e33f043c10744.

> Also, no biggie if the answer is "no" and we want to roll with devstats as a measure of interest...

DevStats is not a sufficient measure of interest, but this seemed to be a reasonable to kickstart the conversation about activity, in absence of other concrete data points.

As @xmudrii mentioned, I think the OBS implementation is a great opportunity to dive in.

Additionally, we need to have a wider discussion about selection and promotion workflow and would welcome feedback on https://github.com/kubernetes/sig-release/pull/1958.

---

@kubernetes/sig-release-leads @kubernetes/release-managers -- Given everyone has checked in, we can merge this ahead of the feedback deadline.

...just needs another `/lgtm`. :)

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Comment by justaugustus** [comment]
Thanks Marko!
/hold cancel

**Comment by saschagrunert** [comment]
We need to sync those changes back to all our repositories.
